README.md

```diff
@@ -125,7 +125,7 @@ Projects that require either of the two features need to explicitly declare the
 <dependency>
 	<groupId>com.microsoft.sqlserver</groupId>
 	<artifactId>mssql-jdbc</artifactId>
-	<version>7.1.4.jre11-preview</version>
+	<version>7.2.0.jre11</version>
```

Comment:
Please check line 93 of the pom file to see if that needs an update, can't comment on this PR.

Reply:
No. that's fine. It says:
`To get the latest preview version of the driver, add the following to your POM file:`